tests/operators/test_agnostic_load_file.py

```diff
@@ -155,7 +154,9 @@ def test_unique_task_id_for_same_path(sample_dag):
             params = {
                 "path": str(CWD) + "/../data/homes.csv",
                 "file_conn_id": "",
-                "output_table": Table(name=OUTPUT_TABLE_NAME, conn_id="postgres_conn"),
+                "output_table": Table(
+                    conn_id="postgres_conn", metadata=Metadata(database="pagila")
```

Comment:
Shouldn't we be using `postgres_conn_pagila` here and not pass any metadata?

Reply:
So we only need that for prepopulated tables, here we are populating the table with new data.